src/main/java/brandkon/Product.java

```diff
@@ -0,0 +1,51 @@
+package brandkon;
+
+import jakarta.persistence.*;
+
+@Entity
+public class Product {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String name;
+    private int price;
+    private int expirationDays;
+    private String imageUrl;
+
+    @ManyToOne
+    private Brand brand;
+
+    public Product(long id, String name, int price, int expirationDays, String imageUrl,Brand brand) {
+        this.id = id;
+        this.name = name;
+        this.price = price;
+        this.expirationDays = expirationDays;
+        this.imageUrl = imageUrl;
+        this.brand = brand;
+    }
+
+    public Product() {
```

Comment:
사소한 거긴 한데, 개발자가 직접 사용할 생성자는 아니라서 public으로 두기보다 접근 가능 단계를 한 단계 낮추어 protected로 두면 개발을 좀 더 잘해 보이는 효과가..
```suggestion
    protected Product() {
```